src/examples/evas/evas-images3.c

```diff
@@ -30,10 +30,10 @@ static const char *commands = \
   "commands are:\n"
   "\tw - write new pixel data to image\n"
   "\ti - print image info\n"
-  "\ta - save noise image to disk (/tmp dir)\n"
+  "\ta - save noise image to disk (TEMP dir)\n"
   "\th - print help\n";
 
-const char *file_path = "/tmp/evas-images2-example.png";
+const char *filename = "evas-images2-example.png";
```

Comment:
This one is kind strange. It don't seems to be used anywhere.

Reply:
Yeah, indeed it isn't used anywhere, but as it is a copy of the same template of its brothers `evas-image[2345].c` I changed here too.